extensions/virtual-threads/runtime/src/main/java/io/quarkus/virtual/threads/VirtualThreadsConfig.java

```diff
@@ -3,32 +3,35 @@
 import java.time.Duration;
 import java.util.Optional;
 
-import io.quarkus.runtime.annotations.ConfigItem;
 import io.quarkus.runtime.annotations.ConfigPhase;
 import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
 
+@ConfigMapping(prefix = "quarkus.virtual-threads")
 @ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
-public class VirtualThreadsConfig {
+public interface VirtualThreadsConfig {
 
     /**
-     * Virtual thread name prefix. If left blank virtual threads will be unnamed.
+     * Virtual thread name prefix. The name of the virtual thread will be the prefix followed by a unique number.
+     * The default value is "quarkus-virtual-thread-".
```

Comment:
I’m not sure why you added that? The generated doc takes care of it.

Reply:
Ah forgot about that, I'll fix in a separate PR